index.html

```diff
@@ -0,0 +1,637 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+
+<head>
+  <meta charset="utf-8">
+  <link rel="stylesheet" href="style.css">
+  <title>Bootstrap Project</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css">
+  <script src="https://kit.fontawesome.com/668d372fa7.js" crossorigin="anonymous"></script>
+</head>
+
+<body>
+  <main>
+    <header>
+      <div class="desktop-navbar">
+        <nav class="first-nav">
+          <i class="fas fa-cloud-sun">27°null, MU > </i>
+          <img src="images/logo.png" alt="logo">
+          <div class="right-nav">
+            <a href="#">SIGN IN</a>
+            <button type="button" class="button" name="button">Subscribe ></button>
+          </div>
+
+        </nav>
+
+        <nav class="low-nav ">
+          <a href="#">U.S |</a>
+          <a href="#">World |</a>
+          <a href="#">Tech & Science |</a>
+          <a href="#">Culture |</a>
+          <a href="#">Newsgeek |</a>
+          <a href="#">Sports |</a>
+          <a href="#">Health |</a>
+          <a href="#">The Debate |</a>
+          <a href="#">Weather </a>
+
+          <span>Search <i class="fas fa-search"></i></span>
+
+        </nav>
+
+      </div>
+      <div class="mobile-navbar">
+        <nav class="navbar  mobile-nav">
+          <div class="container-fluid">
+
+            <div class="navbar-header">
+              <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1" aria-expanded="false">
+                <span class="sr-only">Toggle navigation</span>
+                <span class="icon-bar"></span>
+                <span class="icon-bar"></span>
+                <span class="icon-bar"></span>
+              </button>
+              <a class="navbar-brand" href="#">Newsweek</a>
+            </div>
+
+
+            <div class="collapse navbar-collapse nav-i" id="bs-example-navbar-collapse-1">
+              <ul class="nav navbar-nav">
+                <li class="active"><a href="#">SIGN IN <span class="sr-only">(current)</span></a></li>
+                <li><a href="#">SUBSCRIBE</a></li>
+              </ul>
+              <i class="fas fa-bars fa-lg"></i>
+            </div>
+          </div>
+        </nav>
+
+      </div>
+
+    </header>
+
+    <section>
+      <h1>hi</h1>
+      <div class="container-fluid main">
+        <div class="row d-flex justify-content-between">
+          <div class="col-lg-4  col-md-6  col-sm-12 d-flex featured flex-column">
+            <div class="featured1">
+              <h4 class="heading">FEATURED STORIES</h4>
+              <img src="images/bernie.png" alt="">
+              <p><a href="#">Bernie Sanders on Trump's Coronavirus Response: 'Shut This President Up'</a></p>
```

Comment:
- [ ] You should add some bootstrap classes to these types of titles to match the original version and use it in all the places you see the same type of title (it's repeated all over the site).